sidechains.md

```diff
@@ -21,11 +21,11 @@ Some examples of possible rules are:
 - Transferred coins may have a different denominator (not necessarily a 1:1 scale but 1:N)    
 
 ### Coin Distinction
-In sidechains we make a distinction between native coins and transferred coins (we call it external [is this the best name] coins), this is decided at the genesis block and the consensus rules.  
+In sidechains we make a distinction between native coins and transferred coins (we call it external [is this the best name?] coins). This is decided at the genesis block and enforced in the consensus rules.  
 
-Native coins are mined (or pre-mined) and cannot be transferred out to the parentchain, however two sidechains that are both two way pegs can transfer native coins to each other.   
+**Native coins** are mined (or pre-mined) and cannot be transferred out to the parent chain. However, two sidechains that are both two-way peg can transfer native coins to each other.   
 
-External coins are created at the genesis block but are locked, and can only be unlocked when a proof is presented that some coins have been locked on another sidechain.  
+**External coins** are created at the genesis block but are locked, and can only be unlocked when a proof is presented that some coins have been locked on another sidechain.  So at creation time, the sidechain needs to define which other type of coins are allowed to be transferred in. 
```

Comment:
Not entirely true, its not coin types but more like from what sidechain coins are allowed to be transferred.
By default any sidechain coins are allowed
But I am not sure we need to mention this as its already mentioned in earlier, by limiting which genesis blocks are allowed to send coins.